driver_test.go

```diff
@@ -996,6 +1041,118 @@ func testVariousTypes(t *testing.T, arrow bool) {
 	})
 }
 
+func TestArrowVariousTypes(t *testing.T) {
+	runTests(t, dsn, func(dbt *DBTest) {
+		dbt.mustExec("ALTER SESSION set go_query_result_format = arrow_force")
+		rows := dbt.mustQuery(
+			"SELECT 1.0::NUMBER(30,2) as C1, 2::NUMBER(38,0) AS C2, 't3' AS C3, 4.2::DOUBLE AS C4, 'abcd'::BINARY AS C5, true AS C6")
```

Comment:
can you split into multiple test cases per data type?

Reply:
This was the Arrow complement to the `TestVariousTypes` JSON test. The test per data type can be found under `TestArrow<data type>`. If you feel this test is unnecessary, then I can remove it.

Reply:
ok, can you add a comment explaining the difference?

Reply:
Between `TestVariousTypes` and this one?

Reply:
Done.